GiniVision/Classes/HelpMenuViewController.swift

```diff
@@ -55,49 +85,32 @@ final public class HelpMenuViewController: UITableViewController {
         tableView.rowHeight = tableRowHeight
         tableView.backgroundColor = Colors.Gini.pearl
         
-        // On iOS is .automatic by default and it the transition to this view controller looks weird.
+        // On iOS is .automatic by default, having an initial animation when the view is loaded.
         if #available(iOS 11.0, *) {
             tableView.contentInsetAdjustmentBehavior = .never
         }
+        setupNavigationItem(usingResources: backToCameraButtonResource,
+                            selector: #selector(back),
+                            position: .left,
+                            target: self)
     }
     
-    // MARK: - Table view data source
-    
-    override public func numberOfSections(in tableView: UITableView) -> Int {
-        return 1
-    }
-    
-    override public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return items.count
-    }
-    
-    override public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: helpMenuCellIdentifier, for: indexPath)
-        cell.textLabel?.text = items[indexPath.row].0
-        cell.textLabel?.font = cell.textLabel?.font.withSize(14)
-        cell.accessoryType = .disclosureIndicator
-        cell.backgroundColor = .white
-        
-        return cell
-    }
-    
-    override public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        if let viewController = viewController(forRowWithId: items[indexPath.row].id) {
-            navigationController?.pushViewController(viewController, animated: true)
-        }
+    func back() {
+        navigationController?.popViewController(animated: true)
```

Comment:
Normally, I I wouldn't recommend popping yourself but... I'll allow it here :)

Reply:
Me neither, but since it can be reused as a separated view for the component API it is quite limited.
I would have created a HelpMenuCoordinator for this views.